platform/package.json

```diff
@@ -28,9 +29,11 @@
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-react": "^7.31.1",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "postcss": "^8.4.16",
     "prettier": "2.7.1",
     "pretty-quick": "^3.1.3",
+    "tailwindcss": "^3.1.8",
     "webpack": "^5.74.0"
   },
-  "packageManager": "yarn@3.2.3"
+  "packageManager": "yarn@1.22.19"
```

Comment:
The old package manager was causing package installation issues as CI expects v1.22.19 during `yarn build`.

Reply:
Thanks @Codebmk 
I however notice the deploy preview is still failing. Could you look into it too?

Reply:
@123MwanjeMike yep noticed that but the cause/error is not defined. What does the test workflow do?

Reply:
Hi @Codebmk.
The test workflow runs the UI unit tests, and the deploy preview workflow spins up deploy previews.